package.json

```diff
@@ -58,10 +58,10 @@
     "jest-styled-components": "^7.0.3",
     "lint-staged": "^12.0.2",
     "prettier": "^2.1.2",
-    "snyk": "^1.529.0",
     "storybook-addon-performance": "^0.16.1"
   },
   "dependencies": {
+    "@snyk/protect": "^1.880.0",
```

Comment:
Shouldn't this be in the `dev-dependencies`?

Reply:
yup